src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,91 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [isLoading, setIsLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+  const clearTodo = () => dispatch(currentTodoActions.removeTodo());
+
+  let todoID;
+  let title;
+  let completed;
```

Comment:
It's not a good practice to use `let` in the component. Why didn't you use state for these purposes?

Reply:
Fix